ifrs17/Constants/Validations.ipynb

```diff
@@ -108,92 +73,30 @@
-                "\n    (Error.NoScenarioOpening              , 0) => \"Only Best Estimate is valid Scenario for Openings\",",
-                "\n    ",
-                "\n    // Exchange Rate",
-                "\n    (Error.ExchangeRateNotFound           , 2) => $\"Exchange Rate for {s[0]} {s[1]} is not present.\",",
-                "\n    (Error.ExchangeRateCurrency           , 1) => $\"{s[0]} does not have any Exchange Rate defined.\",    ",
-                "\n    // Data Node State",
-                "\n    (Error.ChangeDataNodeState            , 0) => $\"Data Node State can not change from Inactive state into Active state.\",",
-                "\n    (Error.ChangeDataNodeState            , 1) => $\"Data Node State for {s[0]} can not change from Inactive state into Active state.\",",
-                "\n    (Error.ChangeDataNodeState            , 3) => $\"Data Node State for {s[0]} can not change from {s[1]} state into {s[2]} state.\",",
-                "\n    (Error.InactiveDataNodeState          , 1) => $\"Data imported for inactive Data Node {s[0]}.\",",
-                "\n    //Parameters",
-                "\n    (Error.ReinsuranceCoverageDataNode    , 2) => $\"Invalid Reinsurance Coverage parameter does not link a GroupOfReinsuranceContract to a GroupOfInsuranceContract. Provided GroupOfContracts are: {s[0]}, {s[1]}.\",",
-                "\n    (Error.DuplicateInterDataNode         , 2) => $\"Duplicated Inter-DataNode parameter for {s[0]}-{s[1]} is found.\",",
-                "\n    (Error.DuplicateSingleDataNode        , 1) => $\"Duplicated Single-DataNode parameter for {s[0]} is found.\",",
-                "\n    (Error.MissingSingleDataNodeParameter , 1) => $\"Single DataNode Parameter for Data Node {s[0]} is not found.\",",
-                "\n    (Error.InvalidDataNode                , 1) => $\"Data imported for invalid Data Node {s[0]}.\",",
-                "\n    (Error.InvalidDataNodeForOpening      , 1) => $\"Data imported for invalid Data Node or for a Data Node after its inception year {s[0]}.\",",
-                "\n    (Error.InvalidCashFlowPeriodicity     , 1) => $\"Invalid CashFlowPeriodicity parameter for Data Node {s[0]}.\",",
-                "\n    (Error.MissingInterpolationMethod     , 1) => $\"Missing InterpolationMethod parameter for Data Node {s[0]}.\",",
-                "\n    (Error.InvalidInterpolationMethod     , 1) => $\"Invalid InterpolationMethod parameter for Data Node {s[0]}.\",",
-                "\n    (Error.InvalidEconomicBasisDriver     , 1) => $\"Invalid EconomicBasisDriver parameter for Data Node {s[0]}.\",",
-                "\n    (Error.InvalidReleasePattern          , 1) => $\"Invalid ReleasePattern parameters for Data Node {s[0]}.\",",
+                "public class Warning : ValidationBase{",
+                "\n",
+                "\n    protected const string DefaultMessage = \"Warning not found.\";",
```

Comment:
I would remove this variable and move the message in line 96

Reply:
I would prefer to leave it because I use this variable in inheriting classes (client projects). This is the way to avoid bugs that would stem from the typos. I can define a generic constant with this name, but there should be somewhere an accessible variable with this string value.

Reply:
ok, got it. Thanks. 
I agree let's keep it!